docker/README.md

```diff
@@ -0,0 +1,38 @@
+# Cygnus Containerization
+
+The process of building this application and packing it into a container could be a little long in time terms. Addressing this issue, the process is separated in two steps: Building the jar and packing the cygnus container.
+
+## Build the jar
+
+Run this command from the root of this repository
```

Comment:
Please, add ':'  at the end of the line. Do the same with other lines preceding a command.